articles/terraform/terraform-vm-msi.md

```diff
@@ -0,0 +1,92 @@
+# Terraform with a Linux Viratul Machine on Azure
+
+This article shows you how to use a [Terraform Market Place Image](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/azure-oss.terraform?tab=Overview) to create a `Ubuntu Linux VM (16.04 LTS)` with latest `Terraform` installed and configured using [Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/active-directory/managed-service-identity/overview). This image also configures a remote backend to enable [Remote State](https://www.terraform.io/docs/state/remote.html) management using Terraform. This VM image makes it easy to get started using Terraform on Azure in minutes, without having to install and configure Terraform and configure authentication manually. 
+
+There are no software charges for this data science VM image. You pay only the Azure hardware usage fees that are assessed based on the size of the virtual machine that you provision with the VM image. More details on the compute fees can be found on the VM listing page on the Azure Marketplace.
+
+### Prerequisites
+Before you can create a Linux Terraform Virtual Machine, you must have an Azure subscription. If you do not already have one, see [Create your free Azure account today](https://azure.microsoft.com/free/).  
+
+### Create your Terraform Virtual Machine 
+
+Here are the steps to create an instance of Linux Terraform Virtual Machine. 
+
+1. Navigate to [Create a Resource](https://ms.portal.azure.com/#create/hub) listing on the Azure portal.
+2. Search for `Terraform` in the `Search the Marketplace` search bar. Click on `Terraform` template. Click the `Create` button in the Terraform details blade.
+![alt text](media\terraformmsi.png)
+3. The following sections provide inputs for each of the steps in the wizard (enumrated on the the right) used to create the Terraform Linux Virtual Machine.  Here are the inputs needed to configure each of these steps
```

Comment:
enumerated is misspelled.

Reply:
Thanks fixed. If only there was spell checker in VSCode.